pep-9999.rst

```diff
@@ -0,0 +1,597 @@
+PEP: 9999
+Title: Support for parsing TOML in the Standard Library
+Author: Taneli Hukkinen, Shantanu Jain <hauntsaninja at gmail.com>
+Sponsor: TODO
+PEP-Delegate: TODO
+Discussions-To: TODO
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 01-Jan-2022
+Python-Version: 3.11
+Post-History: 1900-01-01
+
+
+Abstract
+========
+
+This proposes adding a module, ``tomllib``, to the standard library for
+parsing TOML. [1]_
+
+
+Motivation
+==========
+
+The TOML format is the format of choice for Python packaging, as evidenced by
+:pep:`517`, :pep:`518` and :pep:`621`. Including TOML support in the standard
+library helps avoid bootstrapping problems for Python build tools. Currently
```

Comment:
I wonder if we should unfold what the bootstrapping problems are and why vendoring is necessary:

A PEP517 build requires reading TOML metadata but the standard library is unable to parse it. How do you build the first TOML parser if building it requires a TOML parser? How do you build a build frontend for building the TOML parser if building it also requires TOML? etc.